app/client/src/features/events/Questions/QuestionList/useQuestionList.tsx

```diff
@@ -16,6 +16,8 @@ export const USE_QUESTION_LIST_FRAGMENT = graphql`
                 node {
                     id
                     question
+                    isVisible
+                    position
```

Comment:
Do we need position on the client to tell if it's enqueued in the question list?  Want to make sure I understand.

Reply:
I just forgot to remove it, I was originally doing a check on the client until I realized we are only fetching 50 so there is no reason to even fetch the non visible ones. It shouldn't be necessary now so I'll remove it.